cypress/support/cypress-support/src/main.js

```diff
@@ -55,6 +55,22 @@ export default function (module) {
       } else {
         cy.log('Unable to establish a pub/sub connection.');
       }
+      const topic = UTILS.getTopic(
+        UTILS.getEnvVariable(CONSTANTS.FIRST_PARTY_APPID),
+        CONSTANTS.SUBSCRIBE,
+        null,
+        CONSTANTS.TOPIC_FBINTERACTIONS
+      );
+      appTransport.subscribe(topic, UTILS.interactionResults);
+      if (UTILS.getEnvVariable(CONSTANTS.INTERACTIONS_METRICS, false) == true) {
+        cy.startOrStopInteractionsService(CONSTANTS.INITIATED).then((response) => {
+          if (response) {
+            Cypress.env(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, true);
+          }
+        });
+      } else {
+        cy.log(CONSTANTS.INTERACTIONS_SERVICE_NOT_ACTIVE);
+      }
```

Comment:
what is the difference between the interaction service being active and enabled ? 
what if there is no response in line 67